src/lib/ChartConfig/timeSeriesDisplayToChartConfig.ts

```diff
@@ -1,13 +1,14 @@
 import { Axis } from "@/components/TimeSeriesComponent/lib/Axis"
 import { ChartConfig } from "@/components/TimeSeriesComponent/lib/ChartConfig.js"
 import { ChartSeries } from "@/components/TimeSeriesComponent/lib/ChartSeries.js"
-import { TimeSeriesDisplaySubplot, TimeSeriesDisplaySubplotItem } from "@deltares/fews-pi-requests"
 import { cssStyleFromFewsLine, cssStyleFromFewsMarker, chartMarkerFromFews } from "./Styles"
+import { TimeSeriesDisplaySubplot, TimeSeriesDisplaySubplotItem } from "../TimeSeries/types"
 
 export function timeSeriesDisplayToChartConfig(subplot: TimeSeriesDisplaySubplot, title: string): ChartConfig {
   const config: ChartConfig = {
     title: title,
-    xAxis: [],
+    sequence: subplot.items[0].sequence !== undefined ? subplot.items[0].sequence : undefined,

```

Comment:
Assumes there is never more than one item per subplot (which might be true for this use case)

Reply:
will be retrieved from the backend soon